src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1017,9 +1138,9 @@ def generate_FDB_6_default_config(self, filename):
         'expectedResponseCodes': [
             [(400,), (400,)],
```

Comment:
Please ensure that these make sense, given the rest of the configuration. As written, I would expect that the original DB files have both the CBSD and a GWBL within 150 km of each FSS. It seems like the modified GWBL DB has moved (or removed) all GWBLs to be further than 150 km from those FSS stations, thus the FSS+GWBL exclusion zone is lifted. Can you please make sure that the data is consistent with this, or that the expected response codes change?

Reply:
FDB.6     Only the mentioned GWBLs are moved further than 150 km from FSS sites (not   all of them moved).